### ISSUE TITLE
Babel 5 only?

### ISSUE THREAD

**Issue by peisenmann** [babel-plugins/babel-plugin-flow-comments#23]
This plugin didn't work for me with Babel 6, but the very similar https://www.npmjs.com/package/babel-plugin-transform-flow-comments plugin did. I'm wondering if this project is intended for use only with Babel 5, and if so, should that be noted somewhere in the Readme.md?

**Comment by hzoo** [comment]
Yeah this is only babel 5 and the repo you linked is already in the babel/babel repo and ported from this. We moved almost all of those plugins out of this org.
